spec/lib/appsignal_spec.rb

```diff
@@ -283,6 +283,14 @@
       end
     end
 
+    describe ".set_namespace" do
+      it "should do nothing" do
+        expect do
+          Appsignal.set_namespace("custom")
+        end.to_not raise_error
+      end
+    end
```

Comment:
These specs are a bit weird. The whole `expect do .. end.to_not raise_error` doesn't test anything more than just calling the method. If it raises an error the test would fail anyway. No need to fix it now, but I'll put it on my list of things to look at.